src/xgov-dapp/src/features/vote/VoteResults.tsx

```diff
@@ -34,6 +35,28 @@ export const VoteResults = ({
   const [isDownloadingCsv, setIsDownloadingCsv] = useState(false)
   const [error, setError] = useState<string | null>(null)
 
```

Comment:
Could use a feature flag here, `isReserveListEnabled`. This should be used primarily on the render and cloned list. See comments bellow

Reply:
Will handle in a different PR